thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/util/ThirdEyeUtils.java

```diff
@@ -437,6 +439,32 @@ public static String getDataSourceFromMetricFunctions(List<MetricFunction> metri
     return map;
   }
 
+  /**
+   * Prints messages and stack traces of the given list of exceptions in a string.
+   *
+   * @param exceptions the list of exceptions to be printed.
+   * @param maxWordCount the length limitation of the string; set to 0 to remove the limitation.
+   *
+   * @return the string that contains the messages and stack traces of the given exceptions.
+   */
+  public static String exceptionsToString(List<Exception> exceptions, int maxWordCount) {
+    String message = "";
+    if (CollectionUtils.isNotEmpty(exceptions)) {
+      StringBuilder sb = new StringBuilder();
+      for (Exception exception : exceptions) {
+        sb.append(exception.getMessage()).append("\n").append(ExceptionUtils.getStackTrace(exception));
```

Comment:
It only truncate the last exceptions. Can we truncate each of the exception message if the size is too large?